files/Gemfile

```diff
@@ -10,10 +10,14 @@ gem "afmotion" # Networking
 # gem "ProMotion-map", "~> 0.3"  # PM::MapScreen
 #
 # For beta distribution a exception reporting
-#gem "motion-hockeyapp" 
+#gem "motion-hockeyapp"
 
-# Useful tools 
-#gem "bubble-wrap" 
+# Useful tools
+#gem "bubble-wrap"
+
+# Debugging tools
+#gem "motion_print" # Useful REPL output for objects
```

Comment:
Ahh... didn't realize that :)

It feels weird not having it in my Gemfile though!